superset-frontend/src/components/Form/Form.tsx

```diff
@@ -29,7 +29,7 @@ const StyledForm = styled(AntDForm)`
   }
 `;
 
-export default function Form(props: FormProps) {
+export default function Form(props: any) {
```

Comment:
Why remove type checking?

Reply:
> Why remove type checking?

It is causing this error given below in the `Fieldset` component: 

```
Type '{ children: any[]; componentClass: string; className: string; layout: "vertical"; }' is not assignable to type 'IntrinsicAttributes & FormProps<any>'.
 Property 'componentClass' does not exist on type 'IntrinsicAttributes & FormProps<any>'.
```
I wasn't sure how else to handle it. Tried this [solution from stackoverflow](https://stackoverflow.com/a/57312722), but it didn't help. 

Any suggestions are welcome.

Reply:
I think we should keep this type checking. it seems to reveal a bug, as `componentClass="fieldset"` is passed into Form in the `Fieldset`component, but isn't expected on the Form. My guess would be that this does nothing at all and we should remove that prop setting from `Fieldset`, but it's also possible that this is undocumented behavior in antd. I'd recommend looking into how this renders, and see if anything actually has that `fieldset` class on it. If not, then it's probably safe to remove the setting of that prop (since according to TS, it didn't do anything in the first place)

Reply:
Ok,  I ran the grep command to search all the JSX elements that use `fieldset` class. But there seems to be none. So, I guess, we can remove this prop safely and see how it render. Lemme know if I should proceed with it.

```
src git:(enhancement/migrate-jsx-to-tsx_5) grep -r "fieldset" .   
   
./explore/controlUtils/getControlState.ts:    section.controlSetRows.forEach(fieldsetRow =>
./explore/controlUtils/getControlState.ts:      fieldsetRow.forEach((field: CustomControlItem) => {
./CRUD/Fieldset.tsx:      <Form componentClass="fieldset" className="CRUD" layout="vertical">

```

Reply:
yeah, let's remove this prop. since the css class isn't used anywhere, then i think it's safe.

Reply:
ok, removed.